include/wlr/backend/drm.h

```diff
@@ -33,6 +33,21 @@ bool wlr_output_is_drm(struct wlr_output *output);
  */
 uint32_t wlr_drm_connector_get_id(struct wlr_output *output);
 
+/**
+ * Leases a given output to the caller. The output must be from the associated
+ * DRM backend.
+ * Returns -errno on error.
```

Comment:
In general wlroots APIs return a bool. Is there a reason why returning a negative errno is useful here?